src/microbe_stage/AvailableUpgrade.cs

```diff
@@ -75,6 +83,22 @@ public void Resolve()
     {
         if (!string.IsNullOrEmpty(IconPath))
             LoadedIcon = GD.Load<Texture2D>(IconPath);
+
+        // Preload the scene for instantiating in microbes
+        // TODO: switch this to only load when loading the microbe stage to not load this in the future when we have
+        // playable stages that don't need these graphics
+        if (!string.IsNullOrEmpty(overrideGraphics.ScenePath))
+        {
+            loadedSceneData.LoadFrom(overrideGraphics);
+        }
+    }
+
+    public LoadedSceneWithModelInfo TryGetGraphicsScene()
```

Comment:
```suggestion
    public bool TryGetGraphicsScene(out LoadedSceneWithModelInfo overrideGraphics)
```

The signature should be like this to match the other TryGet methods I added, and also to follow the convention set by the C# standard library with its TryGet methods.